### ISSUE TITLE
[BUG] Wrong Highest Grave Decoration

### ISSUE THREAD

**Issue by fachriem** [NetroScript/Graveyard-Keeper-Savefile-Editor#76]
**Bugs on set Highest grave decoration**

the fence grave is perfect but sculpture show the wrong object? i use Better Save Soul DLC option..

**Comment by NetroScript** [comment]
In the Better Save Soul DLC the item `grave_top_sculpt_mrb_5` (`Marble sculpture VI`) was added. I assumed this is an upgrade to the `grave_top_highangel_mrb_1` (`Marble sculpture IV`) of Game of Crone - is this not correct and the Marble Sculpture IV offers more decoration value?

**Comment by DeadElle** [comment]
itemIDs are right, you just need to "remove and replace" the sculpture for getting the right visual (not the one "in construction")

**Comment by Kiwibomber** [comment]
This was already present in the older version.
Then had to dismantle and rebuild the stature once at all graves.

**Comment by Kiwibomber** [comment]
![grafik](https://user-images.githubusercontent.com/18040502/141691870-4380a99b-a8ba-4dc8-8ca4-0c5c52d87bca.png)

**Comment by NetroScript** [comment]
I revisited the decorations because you mentioned it again, and I identified the problem which caused having to destroy and rebuild it. That has been fixed in a hotfix ba73c840529f7fe8954cfbe0c51e85ff2996cbbd in 0.1.24 (together with setting arbitrary skull values for bodies in graves).

If you download exactly the same release again, you will have the fixed version (I replaced the built .zip file).
